### PR TITLE
#400: Ensure one whitespace precedes ( in closure signatures

### LINKED ISSUE #400: [parenthesis-whitespace] rule flags parens preceded by braces
**Tailor Version:** v0.9.0
**Swift Version:** 2.2
**Platform (Mac/Linux/Windows/CI):** Mac
**Installation Method:** Homebrew

The Parenthesis-Whitespace rule should not exist. It makes no sense, especially in closures.

It turns functions that look like this:

```
MyClass.executeClosure { (param) -> retVal in
                   
}
```

to the following:

```
MyClass.executeClosure {(param) -> retVal in
                   
}
```

Looks really ugly. 

I'm disabling the rule, but I hope you consider removing it.

### PR DESCRIPTION
**Pull request by adityatrivedi**
Fixes #400.

### CODE REVIEW

File: src/main/java/com/sleekbyte/tailor/listeners/whitespace/ParenthesisWhitespaceListener.java [hunk 1]
```diff
@@ -39,6 +40,11 @@ public void enterParameterClause(SwiftParser.ParameterClauseContext ctx) {
                 return;
             }
         }
+
+        //
```
Comment (nehasingh2293) [inline src/main/java/com/sleekbyte/tailor/listeners/whitespace/ParenthesisWhitespaceListener.java]: I think you might have forgotten to add a comment here

Comment (adityatrivedi) [inline src/main/java/com/sleekbyte/tailor/listeners/whitespace/ParenthesisWhitespaceListener.java]: Yep! Thanks!

**Comment by a4sriniv** [comment]
👍

**Comment by nehasingh2293** [comment]
👍

**Pull request merged**